cmd/tfctl/main.go

```diff
@@ -0,0 +1,323 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	securejoin "github.com/cyphar/filepath-securejoin"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/weaveworks/tf-controller/tfctl"
+)
+
+var (
+	// BuildSHA is the tfctl version
+	BuildSHA string
+
+	// BuildVersion is the tfctl build version
+	BuildVersion string
+)
+
+func main() {
+	cmd := run()
+	if err := cmd.Execute(); err != nil {
+		os.Exit(1)
```

Comment:
This can use [cobra.CheckErr](https://pkg.go.dev/github.com/spf13/cobra#CheckErr)